rules/windows/process_creation/proc_creation_win_susp_lolbin_non_c_drive.yml

```diff
@@ -0,0 +1,32 @@
+title: Wscript Execution from Non C Drive
+id: 5b80cf53-3a46-4adc-960b-05ec19348d74
+status: experimental
+description: Detects Wscript or Cscript executing from a drive other than C. This has been observed with Qakbot executing from within a mounted ISO file.
+author: Aaron Herman
+date: 2022/10/01
+references:
+    - https://github.com/pr0xylife/Qakbot/blob/main/Qakbot_BB_30.09.2022.txt
+    - https://app.any.run/tasks/4985c746-601e-401a-9ccf-ae350ac2e887/
+tags:
+    - attack.execution
+    - attack.t1059
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_lolbin:
+        Image|endswith:
+            - '\wscript.exe'
+            - '\cscript.exe'
+    selection_exetensions:
+        CommandLine|contains: '.js'
```

Comment:
Perhaps add `.vbs`, `.vbe` and other susp extensions allowed with wscript and cscript

Reply:
Added vbs/vbe